app/main.py

```diff
@@ -1,3 +1,7 @@
 def count_occurrences(phrase: str, letter: str) -> int:
-    # write your code here
-    pass
+    counter = 0
+    for char in phrase:
+        if char.lower() == letter.lower():
+            counter += 1
+    print(counter)
```

Comment:
Instead of printing the counter, consider returning it. This will make the function more versatile and allow the caller to use the result programmatically.